example/hello_world.py

```diff
@@ -93,6 +93,7 @@ def attachments(test):
       # but you can include any metadata fields.
       test_name='MyTest', test_description='OpenHTF Example Test',
       test_version='1.0.0')
+  OutputToJSON = output.json_factory.OutputToJSON
```

Comment:
Name exposition (is that a term?) like this seems better done with an `import...as` to me.